### DISCUSSION TITLE
How to set a PR destination branch different from main/master

### DISCUSSION THREAD

**Discussion by nevarsin** [renovatebot/renovate#11739] [Request Help] [answered]
Hello there,

We have a project on Bitbucket Cloud for which (as a company policy) we adopted the GitFlow approach.
This means that all work done is normally merged against a branch called 'develop' through the usual routine of feature branches.
Every release, then, a repo admin would merge develop against master (and he's the only one allowed to do so).

Same repo, different branch (no forks involved).

The things we're missing with renovatebot then, is the possibility to set (as a config value or as an env var) a destination (target) PR branch which is NOT the default one. In our case it would be 'develop'.

I read the docs multiple times but found no hint on how something like this can be performed.

Thanks in advance,

Ste

**Comment by viceice** [comment]
Your only option is https://docs.renovatebot.com/configuration-options/#basebranches

Renovate will then create a commit ontop the configured branch.

eg:

```json
{
  "baseBranches": ["develop"]
}
```

Now renovate will open all pr's branches against the `develop` branch.

Renovate can't branch of `main` and target `develop`

**Reply by setchy** [reply]
@nevarsin - keep an eye on this PR - may be of use for you https://github.com/renovatebot/renovate/pull/20860

**Comment by nevarsin** [comment]
Ok I redid my tests and I grasp the logic.
It looks like in any case, the onboarding PR is done against the default branch (main/master/whatever) regardless to what is set as baseBranches.
Subsequent PRs are then applied on the specified baseBranch.

This is a bit confusing for a beginner and it may help to clarify it more on the doc.

Thanks for listening to my rants.

Stefano

**Reply by rarkins** [reply]
All docs are open source in this repo and PRs are welcome: https://github.com/renovatebot/renovate/tree/main/docs/usage

**Comment by setchy** [comment, marked as answer]
@nevarsin - a new config option called [bbUseDevelopmentBranch](https://docs.renovatebot.com/self-hosted-configuration/#bbusedevelopmentbranch) was released in version `35.2.0`.
